src/app/(dashboard)/agents/error.tsx

```
@@ -0,0 +1,5 @@
+"use client";
+import { AgentsViewError } from "@/modules/agents/ui/view/agents-view";
+export default function Error() {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)